spec/src/main/asciidoc/jakarta-stl.adoc

```diff
@@ -1024,7 +1024,7 @@ _With a body_
 ....
 
 .*Body Content*
-JSP. The JSP container processes the body
+JSP. The Jakarta Server Pages container processes the body
 content, then the action trims it and processes it further.
```

Comment:
I didn't update this obvious one as I'm not quite sure how we should word it so left it for another day.